test/Android/BreakPointWhenTrueCondition/TestScript.xml

```diff
@@ -0,0 +1,339 @@
+    </Event>
+  </Command>
+  <Command name="eval i">
+    <EvalResult>1</EvalResult>
+    <EvalType>int</EvalType>
+    <EvalName>i</EvalName>
+    <EvalFullName>i</EvalFullName>
+    <EvalIsExpandable>False</EvalIsExpandable>
+  </Command>
+  <Command name="delbps" />
+  <Command name="bp Source1.cpp 9 when_true i&lt;3" />
+  <Command name="go">
+    <Event name="IDebugBreakpointEvent2" expected="True">
+      <functionName>MyClass::Func</functionName>
+      <function>MyClass::Func(MyClass * this)</function>
+      <file>Source1.cpp</file>
+      <line>9</line>
+      <col>0</col>
+    </Event>
+  </Command>
+  <Command name="eval i">
+    <EvalResult>2</EvalResult>
+    <EvalType>int</EvalType>
+    <EvalName>i</EvalName>
+    <EvalFullName>i</EvalFullName>
+    <EvalIsExpandable>False</EvalIsExpandable>
+  </Command>
+  <Command name="delbps" />
+  <Command name="bp Source1.cpp 9 when_true i&lt;=3" />
+  <Command name="go">
+    <Event name="IDebugBreakpointEvent2" expected="True">
+      <functionName>MyClass::Func</functionName>
+      <function>MyClass::Func(MyClass * this)</function>
+      <file>Source1.cpp</file>
+      <line>9</line>
+      <col>0</col>
+    </Event>
+  </Command>
+  <Command name="eval i">
+    <EvalResult>3</EvalResult>
+    <EvalType>int</EvalType>
+    <EvalName>i</EvalName>
+    <EvalFullName>i</EvalFullName>
+    <EvalIsExpandable>False</EvalIsExpandable>
+  </Command>
+  <Command name="delbps" />
+  <Command name="bp Source1.cpp 9 when_true i&gt;5" />
+  <Command name="go">
+    <Event name="IDebugBreakpointEvent2" expected="True">
+      <functionName>MyClass::Func</functionName>
+      <function>MyClass::Func(MyClass * this)</function>
+      <file>Source1.cpp</file>
+      <line>9</line>
+      <col>0</col>
+    </Event>
+  </Command>
+  <Command name="eval i">
+    <EvalResult>6</EvalResult>
+    <EvalType>int</EvalType>
+    <EvalName>i</EvalName>
+    <EvalFullName>i</EvalFullName>
+    <EvalIsExpandable>False</EvalIsExpandable>
+  </Command>
+  <Command name="delbps" />
+  <Command name="bp Source1.cpp 9 when_true i&gt;=7" />
+  <Command name="go">
+    <Event name="IDebugBreakpointEvent2" expected="True">
+      <functionName>MyClass::Func</functionName>
+      <function>MyClass::Func(MyClass * this)</function>
+      <file>Source1.cpp</file>
+      <line>9</line>
+      <col>0</col>
+    </Event>
+  </Command>
+  <Command name="eval i">
+    <EvalResult>7</EvalResult>
+    <EvalType>int</EvalType>
+    <EvalName>i</EvalName>
+    <EvalFullName>i</EvalFullName>
+    <EvalIsExpandable>False</EvalIsExpandable>
+  </Command>
+  <Command name="delbps" />
+  <Command name="bp Source1.cpp 31" />
+  <Command name="bp Source1.cpp 9 when_true i&gt;10" />
+  <Command name="go">
+    <Event name="IDebugBreakpointEvent2" expected="True">
+      <functionName>Func</functionName>
+      <function>Func()</function>
+      <file>Source1.cpp</file>
+      <line>31</line>
+      <col>0</col>
+    </Event>
+  </Command>
```

Comment:
too many repeating, instead you might want to test other data types and arithmetic expressions etc.